cli/pkg/local/app.go

```diff
@@ -94,6 +121,7 @@ func NewApp(ctx context.Context, ver config.Version, verbose, strict, reset bool
 	rtOpts := &runtime.Options{
 		ConnectionCacheSize:          100,
 		MetastoreConnector:           "metastore",
+		CatalogConnector:             "catalog",
```

Comment:
This is weird – there is no connector named `"catalog"` in `systemConnectors`. Would propose just removing `CatalogConnector` altogether from `runtime.Options`